src/LibraryManager.Vsix/Json/TextviewCreationListener.cs

```diff
@@ -150,7 +150,7 @@ private void OnFileSaved(object sender, TextDocumentFileActionEventArgs e)
 
                         if (newManifest != null)
                         {
-                            await RemoveFilesAsync(newManifest).ConfigureAwait(false);
+                            await RemoveFilesAsync(newManifest, new CancellationToken()).ConfigureAwait(false);
```

Comment:
cancellation token needs to be replaced by the one we get from Background Tasks. New PR for this is comming so will leave like this for now.